appeals/web/src/server/appeals/appeal-documents/appeal-documents.validators.js

```diff
@@ -59,6 +59,42 @@ export const validateDocumentDetailsReceivedDateValid = createValidator(
 		.withMessage('Received date must be a valid date')
 );
 
+export const validateDocumentDetailsReceivedDateIsNotFutureDate = createValidator(
```

Comment:
Could you use `createDateInputDateInFutureValidator` from `appeals/web/src/server/lib/validators/date-input.validator.js` for this? If it needs to work differently, and it's likely to be reused, it might be worth moving this out into a similar reusable location

Reply:
I tried moving it out but that was a no go, the documents need to be handled differently as they are contained in an array. This means the custom validator has to iterate through all documents on that page. That will explain why a validity one was created specifically for documents too.